org.eclipse.scanning.event/src/org/eclipse/scanning/event/AbstractQueueConnection.java

```diff
@@ -405,7 +411,7 @@ public boolean remove(U bean, String queueName) throws EventException {
 			QueueBrowser qb = session.createBrowser(queue);
 			@SuppressWarnings("rawtypes")
 			Enumeration  e  = qb.getEnumeration();
-	
+
 			String jMSMessageID = null;
 			while(e.hasMoreElements()) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reduce the total number of break and continue statements in this loop to use at most one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS135)